src/t8_schemes/t8_default/t8_default_line/t8_dline_bits.h

```diff
@@ -247,16 +247,23 @@ t8_dline_vertex_coords (const t8_dline_t *elem, const int vertex, int coords[]);
 void
 t8_dline_vertex_ref_coords (const t8_dline_t *elem, const int vertex, double coordinates[1]);
 
-/** Convert a point in the reference space of a line element to a point in the
+/** Convert points in the reference space of a line element to points in the
  *  reference space of the tree (level 0) embedded in [0,1]^1.
  * \param [in]  elem       Input line.
- * \param [in]  ref_coords The reference coordinate on the line \f$ [0,1]^1 \f$
- * \param [out] out_coords An array of 1 double that
+ * \param [in]  ref_coords The reference coordinates in the line
+ *                         (\a num_coords times \f$ [0,1]^1 \f$)
+ * \param [in]  num_coords Number of coordinates to evaluate
+ * \param [in]  skip_coords Only used for batch computation of prisms.
+ *                          In all other cases 0.
+ *                          Skip coordinates in the \a ref_coords and
+ *                          \a out_coords array.
+ * \param [out] out_coords An array of \a num_coords x 1 x double that
  * 		                     will be filled with the reference coordinates
- *                         of the point on the line.
+ *                         of the points on the line.
  */
 void
-t8_dline_compute_reference_coords (const t8_dline_t *elem, const double *ref_coords, double *out_coords);
+t8_dline_compute_reference_coords (const t8_dline_t *elem, const double *ref_coords, const size_t num_coords,
+                                   const size_t skip_coords, double *out_coords);
 
```

Comment:
Please also write down why skip_coords was introduced

Reply:
It is documented in the docstring above